pkg/reconciler/pipeline/controller.go

```diff
@@ -90,6 +90,9 @@ func (r *pipelineReconciler) Reconcile(ctx context.Context, req ctrl.Request) (c
 			return result, err
 		}
 	}
+	if err := setResourceStatus(r.client, plCopy); err != nil {
```

Comment:
Shall this go to `reconcileNonLifecycleChanges()`, after `pl.Status.SetPhase(pl.Spec.Lifecycle.GetDesiredPhase(), "")`?

Reply:
yes, I can be, I'll move it